pkg/minikube/cluster/cluster.go

```diff
@@ -178,17 +183,32 @@ func UpdateCluster(d drivers.Driver) error {
 }
 
 // GetCreds gets the generated credentials required to talk to the APIServer.
-func GetCreds(h sshAble) error {
+func GetCreds(d drivers.Driver) error {
```

Comment:
Rename to `SetupCerts`?
Also consider moving to `pkg/minikube/cluster/credentials.go`